packages/shared/src/components/modals/bookmark/BookmarkFolderModal.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { ReactElement, useState, type FormEvent } from 'react';
+import classNames from 'classnames';
+import { usePlusSubscription, useViewSize, ViewSize } from '../../../hooks';
+import { Modal, type ModalProps } from '../common/Modal';
+import { TextField } from '../../fields/TextField';
+import {
+  Typography,
+  TypographyColor,
+  TypographyTag,
+  TypographyType,
+} from '../../typography/Typography';
+import { Button, ButtonVariant } from '../../buttons/Button';
+import { DevPlusIcon } from '../../icons';
+import { plusUrl } from '../../../lib/constants';
+import { anchorDefaultRel } from '../../../lib/strings';
+import { LogEvent, TargetId } from '../../../lib/log';
+import { IconSize } from '../../Icon';
+import { ModalHeader } from '../common/ModalHeader';
+import { FolderIcon } from '../../icons/Folder';
+
+// Feel free to delete if defined somewhere else at a later point
```

Comment:
Rather remove it if it's not going to happen it will live here forever :)

Reply:
I will remove it myself in [MI-664](https://dailydotdev.atlassian.net/jira/software/projects/MI/boards/31?selectedIssue=MI-664) as it has the correct type from another WIP branch already :)